<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Имя</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leName">
            <property name="placeholderText">
             <string>Валерий</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Фамилия</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSurname">
            <property name="placeholderText">
             <string>Синёв</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Отчество</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leFathername">
            <property name="placeholderText">
             <string>Евгеньевич</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Пол</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="genderMale">
            <property name="text">
             <string>М</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="genderFemale">
            <property name="text">
             <string>Ж</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Номер группы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leGroupNumber">
            <property name="placeholderText">
             <string>2375</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonCheckConnection">
          <property name="text">
           <string>Проверить, работает ли сервер</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAddUser">
          <property name="text">
           <string>Добавить студента</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSearchInDB">
          <property name="text">
           <string>Искать студентов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSelectAllFromDB">
          <property name="text">
           <string>Показать всех студентов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonDeleteByID">
            <property name="text">
             <string>Удалить студента</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Id студента</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leID">
              <property name="placeholderText">
               <string>8</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table">
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Отчество</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Пол</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер группы</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
